docs/content/users/extend/additional-services.md

```diff
@@ -105,18 +105,32 @@ If you have add-ons that were installed before v1.22, update them with `ddev add
 
 ## Adding Custom Configuration to an Add-on
 
-Sometimes it's necessary to add custom configuration to an add-on. For example, in [`ddev-redis-7`](https://github.com/ddev/ddev-redis-7) the `image` is set to `image: redis:7.2-alpine`. If you wanted to change this to use `7.0-alpine` instead, you would have two choices:
+Sometimes it's necessary to add custom configuration to an add-on. For example, in [`ddev-redis`](https://github.com/ddev/ddev-redis) the `image` is set to `image: redis:${REDIS_TAG:-6-bullseye}` (it is important to have the default value for `$REDIS_TAG`). If you wanted to change this to use `7-bookworm` instead, you would have two choices:
 
-1. Remove the `#ddev-generated` line in `docker-compose.redis-7.yaml` and edit the relevant line.
-2. Add a `.ddev/docker-compose.redis-7_extra.yaml` with the contents:
+1. With DDEV v1.23.5+, the override can be done using a special `.ddev/.env.*` file, where `*` is the name of the add-on, in this case `.ddev/.env.redis` (file is created automatically):
```

Comment:
I think we have a little bit of trouble here. 

The name of an add-on *may* be the same as the name of the *service*. But it may not be. There's nothing that forces that. In fact, in ddev-ioncube, the name is nonstandard... and there is also not a docker-compose.*.yaml, so it's impossible to do a .env anyway. 

But if there *were* a `docker-compose.*.yaml`, it might contain more than one service. 

I think what we probably have here at this point is just a naming problem. I think you're completely handling all *services* by looking and the fully constructed compose yaml. But that may not be the exact name of the add-on. And if a `docker-compose.*.yaml` had more than one service in it (I think some do) then this would mostly work OK?

Reply:
The add-on name matters only in the `ddev add-on get --something=value` command, and as you mention, this is a problem.

And luckily we can make it better and obvious using a new command `ddev dotenv .env.ioncube --flags`, that I described in:
- https://github.com/ddev/ddev/pull/6406#issuecomment-2363869271